### PR TITLE
Feature/cypress homepage

### PR DESCRIPTION
**Pull request by Neil-B-Linton**
Hi Team, Please review this PR. 

## Description of Changes:
* Cypress Homepage_Spec (will add stubbing if needed - stubbing begins on beginning of NewStories, and end of Homepage)
* Need to most updated branch to tag the crisis line for testing features

## Put an 'X' next to all that apply
New Test: [x]
New Feature:[x]

## Before You Submit Ensure
All Tests Pass: [x]
Full Coverage: [x]
Runs Locally: [x]

Thanks!

### CODE REVIEW

File: cypress/e2e/Homepage_spec.cy.js [hunk 1]
```diff
@@ -26,30 +26,43 @@ describe('Homepage', () => {
   it('Should display two buttons to the user', () => {
     cy.get('.navbar').within(() => {
       cy.get('.navbar__main--menu--button').click()
+      cy.get(':nth-child(1) > .navbar__main--menu--link').should('be.visible')
+      cy.get(':nth-child(2) > .navbar__main--menu--link').should('be.visible')
```
Comment (camianderson) [inline cypress/e2e/Homepage_spec.cy.js:30]: good job testing the buttons!

**Comment by camianderson** [comment]
Nice job! 
Next step is to stub the API call and use it to test.

File: cypress/e2e/Homepage_spec.cy.js (hunk 1 above)
Comment (jordan-sullivan) [reply cypress/e2e/Homepage_spec.cy.js:30]: Love the nth child usage here!

**Review by jordan-sullivan — APPROVED** [review]
Thanks for removing the pseudocode! Great work!! -J

**Pull request merged**